pkg/gcv/validator.go

```diff
@@ -311,11 +310,33 @@ func (v *Validator) ReviewUnmarshalledJSON(ctx context.Context, asset map[string
 	if err := v.fixAncestry(asset); err != nil {
 		return nil, err
 	}
+
+	if asset2.IsK8S(asset) {
+		return v.reviewK8SResource(ctx, asset)
+	}
+	return v.reviewGCPResource(ctx, asset)
+}
+
+// reviewK8SResource will unwrap k8s resources then pass them to the cf client with the gatekeeper target.
+func (v *Validator) reviewK8SResource(ctx context.Context, asset map[string]interface{}) ([]*validator.Violation, error) {
+	k8sResource, err := asset2.UnwrapCAIResource(asset)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to convert asset to admission request")
+	}
+	responses, err := v.k8sCFClient.Review(ctx, k8sResource)
+	if err != nil {
+		return nil, errors.Wrapf(err, "GCP target Constraint Framework review call failed")
```

Comment:
K8s instead of GCP, right?

Reply:
haha, yes, the hazard of cut / paste.  fixed.